test/tc_delete_profiles_test.go

```diff
@@ -140,14 +146,37 @@ spec:
 		e.logf("> > Running test case for deleted profiles and pods: %s", testCase.description)
 		profileCleanup := e.writeAndCreate(deleteProfile, "delete-profile*.yaml")
 		defer profileCleanup()
+		e.waitFor("condition=ready", "seccompprofile", deleteProfileName)
 		podCleanup := e.writeAndCreate(fmt.Sprintf(testCase.podManifest, namespace), "delete-pod*.yaml")
 		defer podCleanup()
 		e.waitFor("condition=ready", "pod", deletePodName)
 		e.logf("Ensuring profile cannot be deleted while pod is active")
 		e.kubectl("delete", "seccompprofile", deleteProfileName, "--wait=0")
-		output := e.kubectl("get", "seccompprofile", deleteProfileName)
-		e.Contains(output, "Terminating")
+
+		e.logf("Waiting for profile to be marked as terminating but not deleted")
+		// TODO(jhrozek): deleting manifests as Ready=False, reason=Deleting, can we wait in a nicer way?
```

Comment:
you'd need a specific condition for deleting :confused:

Reply:
The `Deleting` condition is used when setting one, e.g. `status.SetConditions(rcommonv1.Deleting())`, I was just stumped that I couldn't find an equivalent of `GetConditions(Deleting)`